stdlib/data/recipe.lua

```diff
@@ -82,6 +82,7 @@ end
 -- @tparam table ingredients
 -- @tparam string name Name of the ingredient to remove
 local function remove_ingredient(ingredients, name)
+    name = name.name
```

Comment:
maybe do `name = name.name or name` ? The comment says that it should be a string, so either that should be updated, or both a table and string should be valid.